v3/newrelic/transaction.go

```diff
@@ -633,7 +626,7 @@ func (webrequest WebRequest) GetHost() string {
 	return webrequest.Host
 }
 
-func (webrequest WebRequest) GetBody() []byte {
+func (webrequest WebRequest) GetBody() any {
```

Comment:
Hello @aayush-ap, is there a specific reason why this is an any type? Is this function to be used only for the security agent? If there is a specific type we want to make sure any external usage has an expected return value.

Reply:
yes,getBody is only used by the cses-go-agent(security agent).
https://github.com/newrelic/csec-go-agent/blob/main/security_intercept/intercept.go#L748

Reply:
For now this is not possible because csec-go-agent has already released we can make this change in the next release of both agents.
According to me, this will not make an issue because GetBody only used by the csec agent.

Reply:
Alright. I think we're going to put this on ice for now until the changes to the csec-go-agent are in place. We just want to make sure that any public facing API's remain the same so customer who *may be* using the function in their code bases aren't affected by this change